Django_web/manage.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+"""Django's command-line utility for administrative tasks."""
+import os

```

Comment:
Could you please add the `requirements.txt` file for the project?
In the case of existing this file, we can install all requirements via `pip install -r requirements.txt` with versions of packages. Here it's not so critical, but in real projects with a lot of dependencies, it can be essential.

Reply:
Yes, it's helpful. Added.